image-loader/app/lib/imaging/FileMetadataReader.scala

```diff
@@ -119,6 +119,16 @@ object FileMetadataReader {
       case (key, Some(value)) => key -> value
     }
   }
+
+  private val redactionThreshold = 5000
+  val redactionReplacementValue = s"REDACTED (value longer than $redactionThreshold characters, please refer to the metadata stored in the file itself)"
+  private def redactLongFieldValues(imageId:String)(props: Map[String, String]) = props.map {
+    case (fieldName, value) if value.length > redactionThreshold =>
+      logger.warn(s"Redacting '$fieldName' XMP field for image $imageId, as it's problematically long (longer than $redactionThreshold characters")
```

Comment:
XMP or ICC? Or just drop the prefix from msg?

Reply:
ahh great spot, thanks @paperboyo - will fix now